docs/user/contributing/getting-involved.md

```diff
@@ -67,4 +67,4 @@ We're always looking to improve our systems, especially when they're not functio
 
 ## Translations
 
-Assistance is always welcome in translating our projects [to other languages](https://translate.getsol.us)!
+Assistance is always welcome in translating our projects [to other languages](/docs/packaging/translation-instructions)!
```

Comment:
Do you know where `translate.getsol.us` used to point? We could point it to `https://app.transifex.com/solus/`

Reply:
https://web.archive.org/web/20220520013432/https://translate.getsol.us/

The Weblate dashboard apparently. So yeah, the transifex link would be the up-to-date equivalent.

Reply:
DNS change is not straightforward, we'll leave it as is in your revision.